addons/uimanager.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>UI Manager</title>
+
+    <style>
+        body {
+            background-color: lightgray ;
+        }
+        table, tr, th, td {
+            border-collapse: collapse;
+            border: 1px darkgray solid;
+            padding: 5px;
+        }
+    </style>
+
+    <script>
+        document.title=`${window.location.hostname} UI Manager`;
+        const PureManagedPage = {
+            key: 'PureWindow',
+            url: `/purification/`,
+            refreshCount: 1,
+            refreshPeriod: 5 * 60 * 1000, // refresh every 5 minutes
+        };
+
+        let getCurrentTime = () => {
+            const month = ['Jan','Feb','Mar','Apr','May','Jun','Jul','Aug','Sep','Oct','Nov','Dec'];
+            const d = new Date(Date.now());
+            return `${d.getFullYear()}-${month[d.getMonth()]}-${d.getDate()} ${d.getHours()}:${d.getMinutes()}:${d.getSeconds()}`;
+        }
+
+        let launchManagedPage = (ManagedPage) => {
+            // window.focus();
+            console.log(`Page ${ManagedPage.url}), refresh # ${ManagedPage.refreshCount++} at ${getCurrentTime()}`);
+            let pureRefreshTime = document.getElementById("pureRefreshTime");
+            pureRefreshTime.innerHTML = getCurrentTime();
+            if (window[ManagedPage.key]) {
+                window[ManagedPage.key].close(); window[ManagedPage.key] = null;
+            }
+            setTimeout( () => {
+                window[ManagedPage.key] = window.open(ManagedPage.url, "_blank");
+            }, 200);
+        };
+
+        let onStartOrStopPureWindow = (evt) => {
+            if ((evt.currentTarget.textContent === "Stop")) {
+                evt.currentTarget.textContent = "Start";
+
+                if (typeof window[`${PureManagedPage.key}Interval`] === 'number') {
+                    clearInterval(window[`${PureManagedPage.key}Interval`]);
+                    window[`${PureManagedPage.key}Interval`] = null;
+
+                    if (typeof window[`${PureManagedPage.key}`] !== 'undefined') {
+                        window[`${PureManagedPage.key}`].close();
+                        window[`${PureManagedPage.key}`] = null;
+                    }
+                }
+                return;
+            }
+
+            // Start the cycle
+            evt.currentTarget.textContent = "Stop";
+            launchManagedPage(PureManagedPage); // First Time
+            let pureLink = document.getElementById('pureLink');
+            pureLink.innerHTML = `<a.href="${PureManagedPage.url}">Purification</a>`;
+            window[`${PureManagedPage.key}Interval`] =
+                setInterval( "launchManagedPage(PureManagedPage)", PureManagedPage.refreshPeriod);
+        };
+    </script>
+</head>
+<body>
+<table>
+    <tr><th>Tab</th><th>Last Refresh</th><th>Memory</th><th>Action</th></tr>
+    <tr>
+        <td id="pureLink">Purification</td>
```

Comment:
This file has a bunch of purification-specific stuff in it, is there a reason it has to be in this repo?

Reply:
Deleted this file - no longer necessary since I verified that `minutesBeforeAutoPageReload` (new name) works to accomplish the same function.